### PR TITLE
add support for JenkinsPipeline strategy env update

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1509793

Adds support for updating environment variables on buildConfigs
that use the JenkinsPipeline strategy.

/kind bug

cc @openshift/cli-review @bparees

### CODE REVIEW

**Comment by bparees** [comment]
do we have any tests for this?

**Comment by juanvallejo** [comment]
@bparees 
> do we have any tests for this?

Added a test

File: test/cmd/env.sh [hunk 1]
```diff
@@ -21,5 +21,27 @@ os::cmd::expect_success_and_text 'oc set env dc/node --env PASS=x,y=z' 'no longe
 # warning is not printed for variables passed as positional arguments
 os::cmd::expect_success_and_not_text 'oc set env dc/node PASS=x,y=z' 'no longer accepts comma-separated list'
 
+# create a build-config object with the JenkinsPipeline strategy
+os::cmd::expect_success 'oc create -f test/templates/testdata/jenkins-ephemeral.json'
```
Comment (bparees) [inline test/cmd/env.sh]: why can't you use this out of examples/jenkins ?

Comment (juanvallejo) [reply test/cmd/env.sh]: Thanks, switched to using the template in examples/jenkins

**Comment by bparees** [comment]
squash and lgtm

**Pull request merged**